src/defrag.c

```diff
@@ -189,13 +189,13 @@ DefragContextNew(void)
 
     /* Initialize the pool of frags. */
     intmax_t frag_pool_size;
-    if (!ConfGetInt("defrag.max-frags", &frag_pool_size) || frag_pool_size == 0) {
+    if (!ConfGetInt("defrag.max-frags", &frag_pool_size) || frag_pool_size == 0 ||
```

Comment:
can we use a size parsing func with valid range specified?

Reply:
`ConfGetInt` has no variant with range...
Using `StringParseU64RangeCheck` would need more lines of code : first get the string from the config, then parse it and check its range...

This looks simpler, does it not ?